.kokoro/release/snapshot.cfg

```diff
@@ -2,5 +2,5 @@
 
 env_vars: {
   key: "TRAMPOLINE_BUILD_FILE"
-  value: "github/cloud-sql-jdbc-socket-factory/.kokoro/release/snapshot.sh"
-}
+  value: "github/alloydb-java-connector/.kokoro/release/snapshot.sh"
```

Comment:
this should be cloud-sql-jdbc-socket-factory

Reply:
Fixed.